### ISSUE TITLE
[Feature] Enable the ability to not have Authorization header on every example

### ISSUE THREAD

**Issue by damonbanks** [fern-api/fern#2384]
### What can be improved
We have a few Auth Endpoints that dont require an Authorization header. It would be great to have a flag that would remove that header from the example:
`endpoints:
    list:
      path: ""
      method: GET
      auth: false`
This would generate and example that didnt include the Authorization header

**Comment by dsinghvi** [comment]
@damonbanks a little hard to parse the issue, is the issue here that the docs are generating examples with auth for endpoints that don't need it?

**Comment by dsinghvi** [comment]
Closing the issue since `auth: false` seems to solve!

**Comment by damonbanks** [comment]
Yes. And I see that its already a thing. Disregard. Thanks.
